ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/cmd/linux/openstack/inifiles.py

```diff
@@ -161,6 +161,7 @@ def collect(self, device, log):
             'port': device.zCommandPort,
             'user': device.zCommandUsername,
             'password': device.zCommandPassword,
+            'identities': [device.zKeyPath],
```

Comment:
I don't think this will work unless you also add zKeyPath to inifiles.deviceProperties first.